uno.platform.uno-calculator.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: uno.platform.uno-calculator
+runtime: org.gnome.Platform
+runtime-version: '46'
```

Comment:
Does this need the GNOME platform or can freedesktop runtime work?

Reply:
It seems to work with Freedesktop, I changed it